avocado_vt/plugins/vt.py

```diff
@@ -207,7 +207,7 @@ def configure(self, parser):
         add_option(parser=vt_compat_group_qemu,
                    dest='vt.extra_params',
                    arg='--vt-extra-params',
-                   nargs='*',
+                   action='append',
```

Comment:
I think this commit is part of #3095

Reply:
> I think this commit is part of #3095

Yes, that was on purpose, because it was a fix in itself, but it was also needed here.  You can ignore it, as it will be handled fine during a merge.

Reply:
ok